src/components/Pill/PillButton/__tests__/PillButton.spec.tsx

```diff
@@ -94,24 +130,38 @@ describe('<PillButton> component', () => {
         });
 
         it('should render correctly', () => {
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('.PillButton__pill--isActive')).toHaveLength(1);
-            expect(wrapper.find('.PillButton__pill--isOpen')).toHaveLength(0);
+            expect(view.container).toMatchSnapshot();
+            expect(screen.getByRole('button', { name: '' })).not.toHaveClass(
+                'PillButton__button PillButton__button--isOpen'
+            );
+            expect(screen.getByRole('button', { name: 'This pill is in use' })).toHaveClass(
```

Comment:
why to use the text here and not the variable when looking for the name of the button?

Reply:
:+1: